src/components/volunteer/VolunteerHomeSection.tsx

```diff
@@ -85,14 +89,16 @@ export const VolunteerHomeSection = ({
             title={sectionTitle}
           />
         )}
-        <CalendarListToggle showCalendar={showCalendar} setShowCalendar={setShowCalendar} />
+        {calendarToggle && (
+          <CalendarListToggle showCalendar={showCalendar} setShowCalendar={setShowCalendar} />
+        )}
         {showCalendar ? (
           <Calendar
+            isListRefreshing={isLoading}
             query={query}
             queryVariables={queryVariables}
-            calendarData={sectionData}
-            isLoading={isLoading}
             navigation={navigation}
+            additionalData={sectionData}
           />
```

Comment:
I see there is no alphabetical sorting here either.

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/pull/940/commits/f653378eefcca79cef1def80dfa038e995b1bccf